lib/tools/updateInstructions.ts

```diff
@@ -7,27 +8,36 @@ export default function updateInstructions({
 	customerId,
 	repoFullName,
 	issueId,
+	instructionCreator,
+	instructionCommentLink,
 	octokit
 }: {
-	prisma: any
+	prisma: PrismaClient
 	octokit: any
 	issueId: string
 	customerId: string
 	repoFullName: string
+	instructionCreator: string
+	instructionCommentLink: string
 }) {
 	return new DynamicStructuredTool({
 		description:
 			'User will explicitly ask for custom instructions to be updated.',
 		func: async ({newInstructions}) => {
-			const res = await prisma.project.update({
-				where: {
-					customerId_name: {
+			const project = (
+				await prisma.project.findMany({
```

Comment:
Should we do `prisma.project.findUnique({ where: { customerId_name ... } })`? Since instructions are unique to a project

Reply:
good call!